src/main/java/io/singularitynet/MissionHandlers/VideoProducerImplementation.java

```diff
@@ -58,8 +60,10 @@ private AbstractMap.Entry<ByteBuffer, int[]> getRGBFrame()
         Framebuffer framebuffer = MinecraftClient.getInstance().getFramebuffer();
         int i = framebuffer.textureWidth;
         int j = framebuffer.textureHeight;
-        ImageClass imageclass = new ImageClass(i, j, false);
-        ByteBuffer resBuffer = imageclass.getImageAsByteBuffer();
+        NativeImage nativeimage = new NativeImage(i, j, false);
+        ByteBuffer resBuffer = BufferUtils.createByteBuffer(nativeimage.getHeight() * nativeimage.getWidth() * 4);
+        resBuffer.flip();
+        GlStateManager._readPixels(0, 0, nativeimage.getWidth(), nativeimage.getHeight(), nativeimage.getFormat().toGl(), GL11.GL_UNSIGNED_BYTE, resBuffer);
```

Comment:
```java
GlStateManager._readPixels(0, 0, i, j, NativeImage.Format.RGBA.toGl(), GL11.GL_UNSIGNED_BYTE, resBuffer);
```

Reply:
Or RGB - maybe it will be faster

Reply:
If I'm putting RGB there then game will crash each launch of an agent.

So, you want to just remove `NativeImage nativeimage = new NativeImage(i, j, false);` and its usage?